libs/net/dal/Configuration/ProductConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+namespace TNO.DAL.Configuration;
+
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using TNO.Entities;
+
+public class ProductConfiguration : BaseTypeConfiguration<Product, int>
+{
+    public override void Configure(EntityTypeBuilder<Product> builder)
+    {
+        builder.Property(m => m.Id).IsRequired().ValueGeneratedOnAdd();
+        builder.Property(m => m.OwnerId);
```

Comment:
I don't understand the ownership aspect of products here?

Reply:
The owner of the product would be the person who exposed the Report/Notification/etc as a Product.  Might not be the same person who owns it

Reply:
What function does this serve?  A report is owned by someone because it is their report and they maintain it.  However a product is something MMI is offering to users (which can be a report).